packages/npm-dependencies-validation/test/utils/npmUtil.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { spawnCommand, getNPM } from "../../src/utils/npmUtil";
+import { expect } from "chai";
+
+describe("npmUtil unit test", () => {
+  it("spawnCommand failed", () => {
+    return expect(
+      spawnCommand(["ls"], "./non_existing_path")
+    ).to.be.rejectedWith(`spawn ${getNPM()} ENOENT`);
+  });
+
+  it("spawnCommand succeeded with ls", () => {
+    return expect(spawnCommand(["ls", "--depth=0"], "./")).to.be.fulfilled;
```

Comment:
can you add a minimal assertion on its results?

Reply:
should not chai as promised be used with `.eventually` keyword?
- https://github.com/domenic/chai-as-promised

Reply:
Here I just wanted to check that the command succeeded.
.be.fulfilled is also part of the chai-as-promised API

Reply:
its okay, although exit code "0" is a weak assertion